libldac/debian/copyright

```diff
@@ -10,7 +10,8 @@ Copyright: 2003-2017, Sony Corporation
 License: Apache-2
 
 Files: debian/*
-Copyright: 2020 Sourav Das <souravdas142@gmail.com>
+Copyright: 2020 Andrej Shadura <andrewsh@debian.org>,
+           2022 Sourav Das <souravdas142@gmail.com>
```

Comment:
You don't need the comma at the end, this field is free format (https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#copyright-field).